tendrl/node_agent/persistence/servers.py

```diff
@@ -0,0 +1,56 @@
+from tendrl.bridge_common.etcdobj.etcdobj import EtcdObj
+from tendrl.bridge_common.etcdobj import fields
+
+
+class Cpu(EtcdObj):
+    """A table of the cpu, lazily updated
+
+    """
+    __name__ = 'nodes/%s/cpu'
+
+    node_uuid = fields.StrField("node_uuid")
```

Comment:
As we discussed, the field type definitions should come from yaml file. I understand that, it depends on the another patch Tim is working on. This is fine for now, but needs to be implemented that way